pta-intelligent-search-harvester/src/main/java/fi/maanmittauslaitos/pta/search/utils/Region.java

```diff
@@ -1,113 +1,34 @@
 package fi.maanmittauslaitos.pta.search.utils;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.ObjectReader;
-import fi.maanmittauslaitos.pta.search.HarvesterConfig;
-import org.apache.log4j.Logger;
+public interface Region {
+	Double getArea();
```

Comment:
Good idea to make this into a region. However please make the methods public instead of package scope

Reply:
[Oracle](https://docs.oracle.com/javase/tutorial/java/IandI/interfaceDef.html) says that 

> All abstract, default, and static methods in an interface are implicitly public, so you can omit the public modifier.